crates/swc_ecma_codegen/src/config.rs

```diff
@@ -39,6 +44,9 @@ pub struct Config {
 
     #[cfg_attr(feature = "serde-impl", serde(default))]
     pub inline_script: bool,
+
+    #[cfg_attr(feature = "serde-impl", serde(default = "true_by_default"))]
+    pub reduce_escaped_newline: bool,
```

Comment:
Can you document that this is `true` by default?

Reply:
Got